nknd.go

```diff
@@ -375,3 +384,112 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func GetChordID(seeds []string, publickey []byte) ([]byte, error) {
+	id, err := chain.DefaultLedger.Store.GetID(publickey)
+	if err == nil {
+		if len(id) != 0 && !bytes.Equal(id, make([]byte, 32)) {
+			return id, nil
+		}
+	}
+
+	log.Warningf("get chord ID from localnode faild, no chord ID in ledger")
```

Comment:
I think this should be Info instead of warning because it's expected when node first join